src/screens/DetailScreen.js

```diff
@@ -165,10 +165,19 @@ DetailScreen.navigationOptions = ({ navigation, navigationOptions }) => {
             accessibilityLabel="Teilen Taste"
             accessibilityHint="Inhalte auf der Seite teilen"
           >
-            <Icon
-              xml={share(colors.lightestText)}
-              style={headerRight ? styles.iconLeft : styles.iconRight}
-            />
+            {device.platform === 'ios' ? (
+              <Icon
+                name="ios-share"
+                size={26}
```

Comment:
do we need? if it is the same, then you do not need to specify it explicitly.

Reply:
apparently we do need it, if size is not there icons are not aligned.

Reply:
so using `26` it is aligned and using `normalize(26)` it is not?
but this way `normalize` is missing and you cannot ensure alignment always. so you need to try `normalize(xx)` with different values. best would be to try out in different sized devices per simulators.

Reply:
this is really strange , yes without normalize icons are aligned . I tested it on different devices and it's always the same. Settings icon ( home screen) is also with out a normalize , thats why I thought in this case it makes somehow a difference , but I guess not since settings icon is not behaving differently if normalize is there or not ..

Reply:
ok thanks. then keep it as is with `size={26}`